ludwig/error.py

```diff
@@ -47,3 +47,18 @@ def __str__(self):
 
     def __reduce__(self):
         return type(self), (self.column_name, self.feature_type, self.message)
+
+
+@PublicAPI
+class ConfigValidationError(LudwigError, ValueError):
```

Comment:
You need to make this error type serializable. See https://github.com/ludwig-ai/ludwig/pull/2695.

In doing so, please add a unit test similar to the one from the linked PR to verify that serialization works.